package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "http-server": "^0.11.1",
     "node-sass": "^4.7.2",
-    "node-sass-watcher": "^0.5.1"
+    "node-sass-watcher": "^0.5.1",
+    "hoek": "^4.2.1"
```

Comment:
What is this added for? Out of curiosity

Reply:
Not sure how it got there?